libnavigation-base/src/main/java/com/mapbox/navigation/base/logger/Logger.kt

```diff
@@ -9,7 +9,7 @@ interface Logger {
      * @param msg The message you would like logged.
      * @param tr An exception to log
      */
-    fun v(tag: String, msg: String, tr: Throwable? = null)
+    fun v(tag: String? = null, msg: String, tr: Throwable? = null)
```

Comment:
As it's easy to mess up 😅, I think we should make sure to build example in Java and in Kotlin very early on, so that we could see how the API changes would impact both languages. What do you think? In this particular case, we're missing a Kotlin example. I'm interested in 👀 how these default parameters would be used from Kotlin as I believe we'll be forcing users to name them, not sure if we want that.

Reply:
I think one symbol is enough, because it is usual naming of log methods, even in Android SDK Log class used one symbol naming for log methods

Reply:
About examples: I added it in MockNavigationActivity (Java) and in ExampleActivity (Kotlin). I think it is ok to add examples in two different activities, because it is overhead to create new examples for logs... wdyt?